crates/rocksdb/src/lib.rs

```diff
@@ -551,7 +555,7 @@ impl RocksDb {
                     e
                 );
             } else {
-                info!(
+                debug!(
```

Comment:
This eliminate the "shutdown" notice at INFO level on startup, which we would otherwise always print.